lang/src/org/partiql/lang/syntax/SqlParser.kt

```diff
@@ -22,6 +22,7 @@ import org.partiql.lang.errors.ErrorCode.*
 import org.partiql.lang.errors.Property
 import org.partiql.lang.errors.Property.*
 import org.partiql.lang.errors.*
+import org.partiql.lang.eval.builtins.Time.Companion.MAX_PRECISION_FOR_TIME
```

Comment:
*Minor*: If the constant was named `MAX_PRECISION`, one only need to import the `Time` type and refer to it as `Time.MAX_PRECISION` which seems more concise.

Reply:
A wildcard import (`Time.*`) might make this ambiguous.